scripts/swebench_lsp_toolchain.py

```
@@ -0,0 +1,7083 @@
+        "graph_schema_version",
+        "completeness_schema_version",
+        "work_item_schema_version",
+        "validation_evidence_schema_version",
+    ):
+        if type(value[field]) is not int or value[field] <= 0:
+            raise ToolchainError(f"producer {field} must be a positive integer")
+    return dict(value)
+
+
+def structural_cache_identity(rna_binary: Path, checkout: Path) -> dict[str, Any]:
+    completed = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)